src/services/dataManager.js

```
@@ -354,6 +355,96 @@ export class DataManager {
+      delete charData.images;
+    }
+
+    const jsonPayload = {
+      character: charData,
+      skills,
+      specialSkills,
+      equipments,
+      histories,
+    };
+
+    const zip = new JSZip();
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)